builder/frameworks/arduino.py

```
@@ -24,254 +24,988 @@
+        return self._framework_dir
+
+    @property
+    def sdk_dir(self):
+        if self._sdk_dir is None:
+            self._sdk_dir = fs.to_unix_path(
+                join(self.framework_dir,
+                     "tools",
+                     "esp32-arduino-libs",
+                     self.mcu,
+                     "include"
+                )
```

Flake8 (7.2.0)

[error] closing bracket does not match visual indentation

(E124)